lib/msal-browser/FAQ.md

```diff
@@ -428,12 +427,6 @@ pca.loginPopup()
 
 For a full implementation, see the sample: [MSAL.js v2 B2C sample](https://github.com/Azure-Samples/ms-identity-javascript-tutorial/tree/main/1-Authentication/2-sign-in-b2c)
 
-## Why is `getAccountByUsername()` returning null, even though I'm signed in?
```

Comment:
This is still useful info I think. We might just need to update to `getAccount` with username filter